tasks/freeradius-postgresql.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Freeradius additional postgresql system packages
+  when: openwisp2_radius and freeradius_sql.dialect == "postgresql"
```

Comment:
the `when` statements in this file look redundant, because it's also stated in the `import_tasks`, which should be enough, isn't it?

Reply:
I add it because of a habit, because ansible changed it's behavior of `when` some versions ago, but I've removed it here because it seems to work! 😄